src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -292,11 +309,18 @@ def initialize(
         :param epoch: The epoch to initialize the modifier and module at.
             Defaults to 0 (start of the training process)
         :param loggers: Optional list of loggers to log the modification process to
+        :param calibration_dataloader: optional dataloader for running post training
+            quantization with the given model. if present, calibration will be run
+            immediately after quantization is enabled
+        :param calibration_function: optional function to use for calibration of
```

Comment:
add information about the default behavior as well as the expected format of the calibration function